compiler/rustc_lint/src/lints.rs

```diff
@@ -657,14 +657,28 @@ pub struct ForLoopsOverFalliblesSuggestion<'a> {
     pub end_span: Span,
 }
 
+#[derive(Subdiagnostic)]
+pub enum UseLetUnderscoreIgnoreSuggestion {
+    #[note(lint_note)]
+    Note,
+    #[multipart_suggestion(lint_suggestion, style = "verbose", applicability = "maybe-incorrect")]
```

Comment:
And here just reference reference the global message instead of the individual ones.
```suggestion
    #[note(lint_use_let_underscore_ignore)]
    Note,
    #[multipart_suggestion(lint_use_let_underscore_ignore, style = "verbose", applicability = "maybe-incorrect")]
```